resources/install/debian/jitsi-videobridge.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=Jitsi Videobridge
+After=network.target
+
+[Service]
+# allow bind to 80 and 443
+AmbientCapabilities=CAP_NET_BIND_SERVICE
+EnvironmentFile=/etc/jitsi/videobridge/config
+Environment=LOGFILE=/var/log/jitsi/jvb.log
+User=jvb
+RuntimeDirectory=jitsi-videobridge
+RuntimeDirectoryMode=0750
+PIDFile=/var/run/jitsi-videobridge/jitsi-videobridge.pid
+# more threads for this process
+TasksMax=65000
+# allow more open files for this process
+LimitNPROC=65000
+LimitNOFILE=65000
+ExecStart=/bin/bash -c "exec /usr/share/jitsi-videobridge/jvb.sh --host=${JVB_HOST:-localhost} --domain=${JVB_HOSTNAME} --port=${JVB_PORT} --secret=${JVB_SECRET} ${JVB_OPTS} < /dev/null >> ${LOGFILE} 2>&1"
+ExecStartPost=/bin/bash -c "systemctl show -p MainPID jitsi-videobridge.service 2>/dev/null | cut -d= -f2 > /var/run/jitsi-videobridge/jitsi-videobridge.pid"
```

Comment:
Doesn't systemd automatically get the right PID since we exec twice?

Reply:
Yes, the pid is correct.

Reply:
So, why do we need to do this dance to put it into the file then?

Reply:
Cause systemd does not write pid files, it is up to the underlying scripts/software to do that.

Reply:
Doh. I see. At any rate, I found about `$MAINPID`: https://medium.com/@jbriggs_24705/you-should-be-using-pidfile-and-mainpid-instead-of-pkill-1935e4531931